index.js

```diff
@@ -17,7 +17,8 @@ module.exports = num => {
 	}
 
 	const exponent = Math.min(Math.floor(Math.log10(num) / 3), UNITS.length - 1);
-	const numStr = Number((num / Math.pow(1000, exponent)).toPrecision(3));
+	const numTmp = Number((num / Math.pow(1000, exponent)).toPrecision(3));
+	const numStr = options && options.localize ? numTmp.toLocaleString() : numTmp;
```

Comment:
Maybe do `options = Object.assign({}, options)` after the `Number.isFinite(num)` if statement at the start of the function.

Reply:
Note that Object.assign required Node.js 5 or greater which breaks compatibilty on the current minimum version 4. I'd just to `opts = opts || {}` on the start of the function.

Reply:
@silverwind `Object.assign` is supported on Node.js 4

Reply:
Uh, you're right, carry on. (I was misinterpreting http://node.green/)

Reply:
Sorry for the late reply. No problem, I'll add `object.assign`.